File: python/674_最长连续递增序列_easy.py
class Solution:
    def findLengthOfLCIS(self, nums: List[int]) -> int:
        if not nums:
            return 0
        cur = 1
        res = 1
        for i in range(1, len(nums)):
            if nums[i] > nums[i-1]:
                cur += 1
            else:
                cur = 1
            res = max(res, cur)
        return res

File: cpp/674_最长连续递增序列_easy.cpp
class Solution {
public:
    int findLengthOfLCIS(vector<int>& nums) {
        int n = nums.size();
        if(!n)
            return 0;
        int res = 1, cur = 1;
        for(int i=1; i<n; i++){
            if(nums[i]>nums[i-1])
                cur++;
            else
                cur = 1;
            res = max(res, cur);
        }
        return res;

    }
};